Lib/smtplib.py

```diff
@@ -942,9 +945,10 @@ def send_message(self, msg, from_addr=None, to_addrs=None,
             to_addrs = [a[1] for a in email.utils.getaddresses(addr_fields)]
         # Make a local copy so we can delete the bcc headers.
         msg_copy = copy.copy(msg)
+        policy = msg.policy.clone()
+        mail_options = mail_options[:]
```

Comment:
Move this line up to the top of the function, please.  I'm not sure how those default argument values got pass the original review; I thought there was an issue to fix them but I can't find it.

Reply:
👍